universal-login-relayer/lib/core/services/messages/IExecutionQueue.ts

```diff
@@ -0,0 +1,10 @@
+import {SignedMessage} from '@universal-login/commons';
+import {QueueItem} from '../../models/QueueItem';
+import Deployment from '../../models/Deployment';
+
+export interface IExecutionQueue {
+  addMessage (item: SignedMessage) : Promise<string>;
+  addDeployment (item: Deployment) : Promise<string>;
```

Comment:
I guess we could unify this method in the future, and make both SignedMessage and Deployment IExecution or sth similar